product_route_profile/models/route_profile.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class RouteProfile(models.Model):
+    _name = "route.profile"
+
+    name = fields.Char("Name")
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        default=lambda self: self.env.company.id,
+        required=False,
+        string="Company",
+    )
+    route_ids = fields.Many2many(
+        "stock.location.route",
+        string="Routes",
```

Comment:
I think, for consistency, you should add the domain : `domain=[('product_selectable', '=', True)]` like it is on the original `route_ids` field on template

Reply:
Good point ! I add it.